<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingTemplatesDialog</class>
 <widget class="QDialog" name="BuildingTemplatesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Templates</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Templates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="templatesList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="toolBarLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Template</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="tilesList">
          <item>
           <property name="text">
            <string>Exterior wall</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Door</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Door frame</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Window</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Curtains</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stairs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Roof Cap</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Roof Slope</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Roof Top</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="tileLabel">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string>Tile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="chooseTile">
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Rooms</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="editRooms">
           <property name="text">
            <string>Edit rooms...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add new template</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected template</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/stock-duplicate-16.png</normaloff>:/images/16x16/stock-duplicate-16.png</iconset>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="toolTip">
    <string>Duplicate selected template</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/go-up.png</normaloff>:/images/16x16/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move selected template up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/go-down.png</normaloff>:/images/16x16/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="toolTip">
    <string>Move selected template down</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/document-import.png</normaloff>:/images/16x16/document-import.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import templates from a file</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/document-export.png</normaloff>:/images/16x16/document-export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export templates to a file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../tiled.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuildingTemplatesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuildingTemplatesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
